content/2015.05.cliquet-permissions-en.rst

```diff
@@ -0,0 +1,535 @@
+Handling permissions
+####################
+
+:date: 2015-05-12
+:summary: Where we clarify the vocabulary around permissions and explain what
+          we plan to implement for our generic data storage solution, Kinto.
+:slug: la-gestion-des-permissions
```

Comment:
This is due to how pelican handles translations (it adds a `-en` at the end of it) There might be a way to solve this, but not part of this PR.